beoordelingstool.py

```diff
@@ -241,55 +234,25 @@ def unload(self):
     def run(self):
         """Run method that loads and starts the plugin"""
 
-        manholes_layerList = QgsMapLayerRegistry.instance().mapLayersByName("manholes")
-        pipes_layerList = QgsMapLayerRegistry.instance().mapLayersByName("pipes")
-        measuring_stations_layerList = QgsMapLayerRegistry.instance().mapLayersByName("measuring_points")
+        manholes_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_MANHOLES)
+        pipes_layerList = QgsMapLayerRegistry.instance().mapLayersByName(SHP_NAME_PIPES)
```

Comment:
Is `layerList` een qgis begrip? Je mixt underscores en camelcase, vandaar. Ik neem aan dat het op jouw manier het beste past?